systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -1460,6 +1463,69 @@ void checkStorageSizeForVolumes(List<PersistentVolumeClaim> volumes, String[] di
         }
     }
 
+    @Tag(SCALABILITY)
+    @Test
+    void testBigAmountOfTopics() {
+        int numberOfTopics = 300;
+
+        testMethodResources().kafkaEphemeral(CLUSTER_NAME, 3, 1).done();
+
+        for (int i = 0; i < numberOfTopics; i++) {
+            String topicName = "topic-example" + i;
+            testMethodResources().topic(CLUSTER_NAME, topicName, 3, 3).done();
+            LOGGER.info("Checking that topic {} was created", topicName);
+            assertThat(cmdKubeClient().list("kafkatopic"), hasItems(topicName));
```

Comment:
This just seems to be checking that after creating a `KafkaTopic` that it really still exists in k8s. That doesn't prove that the TO has done anything with it. We should check that the topic actually exists in Kafka. 

We should also test that the TO can cope if all of these topics are modified and deleted.  And perhaps not just from the Kubernetes side, but also doing these things using the Kafka admin client.

Reply:
I created two scenarios 
- creating topics via K8s and verifying with Kafka  ->  editing via K8s and verifying via Kafka
- creating topics via Kafka and verifying with K8s  ->  editing via Kafka and verifying via K8s
- deletion of topics are automatically done after test